astropy/coordinates/tests/accuracy/test_ecliptic.py

```diff
@@ -25,7 +25,7 @@ def test_against_pytpm_doc_example():
     astropy_out = fk5_in.transform_to(pytpm_out)
 
     # we check w/i 1 arcmin because there are some subtle differences in pyTPM's ecl definition
```

Comment:
Can you update the comment (or maybe just remove it)?

Reply:
Removed!